app/src/main/java/sdp/moneyrun/ui/MainActivity.java

```diff
@@ -17,12 +17,15 @@
 public class MainActivity extends AppCompatActivity {
     public final static String TAG = MainActivity.class.getSimpleName();
     private TextView mTextView;
-
+    public static boolean calledAlready = false;
```

Comment:
Fundamentally it doesn't change much but isn't better to have a private attribute that we get and change through a getter and a setter?

Reply:
I think that for a static variable it is ok